tests/integration-basic/tests.js

```diff
@@ -86,26 +114,31 @@ describe('Service Lifecyle Integration Test', function() {
     // eslint-disable-next-line no-unused-expressions
     expect(timestamp).to.not.undefined;
 
-    execSync(`${serverlessExec} rollback -t ${timestamp}`, { cwd: tmpDir });
+    await spawn(serverlessExec, ['rollback', '-t', timestamp], { cwd: tmpDir, env });
 
-    const invoked = execSync(`${serverlessExec} invoke --function hello --noGreeting true`, {
-      cwd: tmpDir,
-    });
-    const result = JSON.parse(Buffer.from(invoked, 'base64').toString());
+    const { stdoutBuffer: invoked } = await spawn(
+      serverlessExec,
+      ['invoke', '--function', 'hello', '--noGreeting', 'true'],
+      spawnOptions
+    );
+    const result = JSON.parse(invoked);
     // parse it once again because the body is stringified to be LAMBDA-PROXY ready
     const message = JSON.parse(result.body).message;
     expect(message).to.be.equal('Go Serverless v1.0! Your function executed successfully!');
   });
 
-  it('should remove service from aws', () => {
-    execSync(`${serverlessExec} remove`, { cwd: tmpDir });
-
-    return CF.describeStacks({ StackName })
-      .promise()
-      .then(d => expect(d.Stacks[0].StackStatus).to.be.equal('DELETE_COMPLETE'))
-      .catch(error => {
-        if (error.message.indexOf('does not exist') > -1) return BbPromise.resolve();
-        throw new Error(error);
-      });
+  it('should remove service from aws', async () => {
+    await spawn(serverlessExec, ['remove'], { cwd: tmpDir, env });
+
+    const d = await (async () => {
+      try {
+        return await CF.describeStacks({ StackName }).promise();
```

Comment:
Why not `return` without an `await`?

Reply:
It's to catch promise rejection. Without `await` only sync crashes will be caught